sdk-tests/src/test/java/io/dapr/it/configuration/grpc/ConfigurationClientIT.java

```diff
@@ -154,4 +214,22 @@ private static void executeDockerCommand(String[] command) {
             e.printStackTrace();
         }
     }
+
+    private static void executeDockerCommand(String key, int postfix) {

```

Comment:
almost repetition of method `executeDockerCommand(String[] command)`. Please change to use only one format.